portfolio/src/main/webapp/home-styles.css

```diff
@@ -0,0 +1,126 @@
+*{
+    margin:0;
+    padding: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+body{
+    background-color: #2B2D42;
+}
+/* COLOR PALLETTE:
+Black: 000000
+White: FFFFFF
+Yellow: F8F32B
+Gray: 8D99AE
+Dark blue: 2B2D42
+
+*/
+
+.content-container{
+    position: relative;
+    top:20vh;
+
+}
+.header-container{
+    width: 50%;
+    margin:auto;
+    top: 1000px;
+    color: #EFEBD8;
+}
+.title{
+    font-size: 100px;
+    font-family: Helvetica;
+    
+}
+
+
+.search-container{
+    
+    top: 10vh;
+    border: 1px solid grey;
+    margin: 8px 0;
+    padding: 12px 18px;
+    margin-top: 50px;
+    border-radius: 8px;
+    z-index: 1001;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+    background-color: #FFFFFF;
+    
+}
+
+.search-container label{
+    display: block;
+    text-align: left;
+    color: #333;
+    text-transform: uppercase;
+    font-size: 14px;
+}
+
+.search-container input, .search-container textarea{
+    width: 100%;
+    border: none;
+    background: none;
+    outline: none;
+    font-size: 14px;
+    margin-top: 6px;
+    
+}
+
+.topic-container{
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+    padding-top: 200px;
+    padding-bottom: 100px;
+
+}
+
+.topic-card{
+    width: 25vw;
+    height: 80vh;
+    background-color: #EFEBD8;
+    border-radius:6px;
+}
+.topic-card p{
+    padding: 5px;
+}
+.topic-img-container{
+    width:100%;
+    height:70%;
+}
+
+.topic-img-container img{
+width: 100%;
+height: 100%;
+}
+
+
+
+
+
+/** Taken from https://www.bestcssbuttongenerator.com/#/37**/
+.topic-btn {
+	box-shadow:inset 0px 1px 0px 0px #ffffff;
+	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
+	background-color:#ededed;
+	border-radius:6px;
+	border:1px solid #dcdcdc;
+	cursor:pointer;
+	color:#777777;
+	font-family:Arial;
+	font-size:15px;
+	font-weight:bold;
+	padding:6px 24px;
+	text-decoration:none;
+    text-shadow:0px 1px 0px #ffffff;
+    left: 57%;
+    top: 3%;
+    position: relative;
+
+
+}
+.topic-btn:hover {
+	background:linear-gradient(to bottom, #dfdfdf 5%, #ededed 100%);
+	background-color:#dfdfdf;
+}
```

Comment:
Info comment:D I will try to use this css file to styling the input-form for the page have style consistent

Reply:
Yeah, feel free to change it up however you'd like, I'm not exactly the best when it comes to colors and style and such